administration/src/application/components/forms/OrganizationForm.tsx

```diff
@@ -15,16 +15,42 @@ import {
 
 const organizationCategoryOptions = {
   items: [
-    'Soziales/Jugend/Senioren',
-    'Tierschutz',
-    'Sport',
-    'Bildung',
-    'Umwelt-/Naturschutz',
-    'Kultur',
-    'Gesundheit',
-    'Katastrophenschutz/Feuerwehr/Rettungsdienst',
-    'Kirchen',
-    'Andere',
+    {
```

Comment:
for region i need `item: { label: string, value: string}` so i also adjusted here. we could also provide different Option types